### PR TITLE
Added useBootstrap4 as an initialization parameter

### PR DESCRIPTION
**Pull request by timominous**
Adding the `useBootstrap4` parameter so that it could be set through initialization or through the config file.

### CODE REVIEW

File: Sources/SteamPress/Provider.swift [hunk 1]
```diff
@@ -62,20 +63,24 @@ public struct Provider: Vapor.Provider {
         if let blogPathFromConfig = config[Provider.configFilename, "blogPath"]?.string {
             blogPath = blogPathFromConfig
         }
-        
-        self.init(postsPerPage: postsPerPage, blogPath: blogPath)
+
+        let useBootstrap4: Bool = config[Provider.configFilename, "paginator", "useBootstrap4"]?.bool ?? true
```
Comment (0xTim) [inline Sources/SteamPress/Provider.swift]: You don't need to specify the type (`:Bool`) here as it will infer it

**Review by 0xTim — CHANGES_REQUESTED** [review]
Just a quick style change, other than that it looks really good!

**Review by 0xTim — APPROVED** [review]

**Comment by 0xTim** [comment]
Could you try removing the Package.pins from your branch and removing it from the .gitignore - we probably don't want it in a library and I think that's the issues for the CI errors

**Comment by timominous** [comment]
@0xTim Removed Package.pins but it wasn't on .gitignore. We'll see if the CI finishes up.

**Comment by 0xTim** [comment]
Sorry meant adding it to the `.gitignore`! It shouldn't be in there and it will stop it getting checked in

**Comment by timominous** [comment]
@0xTim Sure. Will add it to .gitignore now.

**Comment by 0xTim** [comment]
Awesome looks like that has fixed the tests

**Pull request merged**